src/gpodder/plugins/soundcloud.py

```diff
@@ -73,12 +82,18 @@ def urlopen_with_token(url):
     else:
         url_without_id = url + "?client_id="
```

Comment:
This could be `url = url + "?client_id=" + client_id`, or `url = "%s?client_id=%d".format(url, client_id)`, and same for line above it, just with the `&`. The other code in this function can then use `url` instead of all the noisy repeated formatting.